package.json

```diff
@@ -20,18 +21,19 @@
     }
   },
   "peerDependencies": {
-    "@adminjs/design-system": ">=3.0.0",
-    "adminjs": ">=6.0.0"
+    "@adminjs/design-system": "^3.0.4",
+    "adminjs": "^6.4.1"
```

Comment:
Changed from `>=6.0.0` to `^6.4.1` because:
a) it depends on the recent changes from 6.x
b) version 7 might contain breaking changes to bundling so doing `^6.x.x` and then `^6.x.x | ^7.0.0` is safer